suse_migration_services/fstab.py

```diff
@@ -103,13 +111,32 @@ def export(self, filename):
         """
         with open(filename, 'w') as fstab:
             for entry in self.fstab:
-                fstab.write(
-                    '{0} {1} {2} {3} 0 0{4}'.format(
-                        entry.device, entry.mountpoint,
-                        entry.fstype, entry.options,
-                        os.linesep
+                if self._is_on_root(entry.device):
+                    fstab.write(
+                        '{0} {1} {2} {3} 0 0 {4}{5}'.format(
+                            entry.device, entry.mountpoint,
+                            entry.fstype, entry.options,
+                            entry.unix_device, os.linesep
+                        )
                     )
-                )
 
     def get_devices(self):
-        return self.fstab
+        return [
+            entry for entry in self.fstab if self._is_on_root(entry.device)
+        ]
+
+    def _get_root_disk(self):
+        """Get the disk name for / partition."""
+        for entry in self.fstab:
+            if entry.mountpoint == '/':
+                self.root_disk = Command.run(
+                    ['lsblk', '-no', 'pkname', entry.unix_device]
+                ).output
+                break
+
+    def _is_on_root(self, device_path):
+        """Check that the entry belongs to the root disk."""
+        lsblk_call = Command.run(
+            ['lsblk', '-n', '-o', 'pkname', device_path]
```

Comment:
just for consistency we should decide for '-n', '-o' or '-no' in all cases. I vote for the shorter form